config.json

```diff
@@ -1,6 +1,7 @@
 {
   "slug": "rust",
   "language": "Rust",
+  "ignore_pattern": "example|description.md|metadata.yml",
```

Comment:
fun fact: this will match a file name descriptionxmd, but we have no files of that name, so I don't care. it will go away if files move to .meta.